wandb/sdk/wandb_init.py

```diff
@@ -345,44 +345,74 @@ def getcaller():
 
 def init(
     job_type: Optional[str] = None,
-    dir=None,
+    dir: Optional[str] = None,
     config: Union[Dict, None] = None,  # TODO(jhr): type is a union for argparse/absl
     project: Optional[str] = None,
     entity: Optional[str] = None,
     reinit: bool = None,
     tags: Optional[List] = None,
-    team: Optional[str] = None,
+    team: Optional[str] = None,  # LB What is this?
```

Comment:
I believe it's an alias for `entity`.  @raubitsj are we using this?

Reply:
ill remove for now, we can add that later